frontend/src/store/modules/website/actions.js

```diff
@@ -8,40 +9,11 @@ export default {
     [SAVE_NEW_WEBSITE]: (context) => {
         const newDataSite = context.state.newWebsite;
 
-        return new Promise((resolve, reject) => {
-            if(!newDataSite.name || !newDataSite.domain) {
-                reject({
-                    errors: {
-                        message: "Fill in all thi fields correctly."
-                    }
-                });
-            }
-            const fakeWebsiteData = {
-                name: 'Website name',
-                domain: 'http://domain.com',
-                single_page: newDataSite.single_page,
-                tracking_number: '123456789-10',
-            };
+        return addWebsite(newDataSite)
```

Comment:
Add check before data saving, that all needed data are ready to be saved, see prev mocked error throwing example for error types